### PR TITLE
Add crowdin workflow

### PR DESCRIPTION
**Pull request by NotMyFault**
This is how the preview looks like:

Bildschirmfoto 2022-05-06 um 13 35 20

### CODE REVIEW

File: workflow-templates/crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        with:
+          upload_translations: false
+          download_translations: true
+          skip_untranslated_files: true
+          push_translations: true
+          export_only_approved: true
+          commit_message: 'feat: New Crowdin translations'
+          create_pull_request: true
+          pull_request_title: 'New Crowdin translations'
+          pull_request_labels: 'localization'
```
Comment (timja) [inline workflow-templates/crowdin.yml:35]: may need a script to run on the org to add this everywhere? as it won't exist in most repositories and then won't get labelled, or a step in the docs?

Comment (NotMyFault) [reply workflow-templates/crowdin.yml:35]: > or a step in the docs?

That is the plan. We could add it as default label to newly created repos in the org, but I don't think we need to force it onto every repository, especially if these don't use release drafter at all.

**Comment by NotMyFault** [comment]
@timja Could we merge this nevertheless to have it available to show during the online meetup? I'll fill in the todo once https://github.com/jenkins-infra/jenkins.io/pull/5140 is merged

**Comment by NotMyFault** [comment]
All set now 👏🏻

**Review by MarkEWaite — APPROVED** [review]
Approving in hopes that we can use it in the webinar today

**Pull request merged**